tests/transport_test.go

```diff
@@ -140,23 +143,26 @@ var _ = Describe("Transport Tests", func() {
 	registryNoAuthEp := func() string { return fmt.Sprintf("docker://%s", utils.RegistryHostName+"."+f.CdiInstallNs) }
 	registryAuthEp := func() string { return fmt.Sprintf("docker://%s.%s:%d", utils.RegistryHostName, f.CdiInstallNs, 1443) }
 	altRegistryNoAuthEp := func() string { return fmt.Sprintf("docker://%s.%s:%d", utils.RegistryHostName, f.CdiInstallNs, 5000) }
+	trustedCriRegistryEp := func() string { return "docker://registry:5000" }
```

Comment:
This should be configurable.  Not exactly sure the best way to do it for func tests but `DOCKER_PREFIX` is the env var used for build/sync.

Reply:
Done.